src/io.c

```diff
@@ -1929,6 +1990,28 @@ static Obj FuncPRINT_FORMATTING_CURRENT(Obj self)
     return output->format ? True : False;
 }
 
+// flush pending output on stdout/errout, and ensure we are
+// at the start of a new line. also reset any formatting hints
+static Obj FuncFLUSH_STDOUT_ERROUT(Obj self)
```

Comment:
This name is not quite accurate; we flush the output buffer, but we *also* move to the next line (but *only* if there is anything on the current line). If anybody has a better suggestion...